src/test/resources/features/list.feature

```diff
@@ -5,4 +5,4 @@ Feature: List actions
     Given I log in as admin user
     When I select TestBoard board
     And I create a new list with "TestList" as title
-    Then I should see the new created list with the given title
\ No newline at end of file
+    Then I should see the new created list with the given title
```

Comment:
Please add an empty line at the end of the class.

Reply:
Solved